.github/workflows/docs.yml

```diff
@@ -0,0 +1,22 @@
+name: docs
+on:
+  push:
+    branches:
+      - main
+permissions:
+  contents: write
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
```

Comment:
Why not use v4?

Reply:
I just used the docs from the material for mkdoc site .. will update